packages/dom/src/feature-app-container.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  FeatureAppDefinition,
+  FeatureAppManager,
+  FeatureAppScope
+} from '@feature-hub/core';
+import {LitElement, html, property} from 'lit-element';
+import {TemplateResult} from 'lit-html';
+
+/**
+ * A DOM Feature App allows the use of any frontend technology such as Vue.js,
+ * React, or Angular.
+ */
+export interface DomFeatureApp {
+  /**
+   * @param container The container element to which the Feature App can attach
+   * itself.
+   */
+  attachTo(container: Element): void;
+}
+
+export function createFeatureAppContainer(
+  manager: FeatureAppManager
+): typeof HTMLElement {
+  class FeatureAppContainer extends LitElement {
+    @property({type: Object})
+    public featureAppDefinition?: FeatureAppDefinition<DomFeatureApp>;
+
+    @property({type: String})
+    public idSpecifier?: string;
+
+    private featureAppScope: FeatureAppScope<DomFeatureApp> | undefined;
+
+    public render(): TemplateResult {
+      return html`
+        ${this.getFeatureApp()}
+      `;
+    }
+
+    public disconnectedCallback(): void {
+      if (this.featureAppScope) {
+        this.featureAppScope.destroy();
+      }
+
+      super.disconnectedCallback();
+    }
+
+    private getFeatureApp(): HTMLElement | TemplateResult {
```

Comment:
I think we can move the entire content of this method to render. I don't like the name and see no reason for outsourcing.